src/SearchResults/index.js

```diff
@@ -133,7 +133,7 @@ var SearchResults = function( state, algoliaResponse ) {
     forEach( result.facets, function( facetResults, dfacet ) {
       var position = disjunctiveFacetsIndices[ dfacet ];
 
-      var dataFromMainRequest = mainSubResponse.facets[ dfacet ];
+      var dataFromMainRequest = mainSubResponse.facets && mainSubResponse.facets[ dfacet ] || {};
```

Comment:
In the response we get from algolia, the values of a single facet is an object of the form object<attributeValue : string, count: number>  https://www.algolia.com/doc#FacetNavigation . Thus it has to be consistent with the object merge we do two lines after https://github.com/algolia/algoliasearch-helper-js/blob/fix/72/src/SearchResults/index.js#L139